src/libs/actions/Task.js

```diff
@@ -479,6 +479,11 @@ function editTaskAssigneeAndNavigate(report, ownerAccountID, assigneeEmail, assi
     // If we make a change to the assignee, we want to add a comment to the assignee's chat
     // Check if the assignee actually changed
     if (assigneeAccountID && assigneeAccountID !== report.managerID && assigneeAccountID !== ownerAccountID && assigneeChatReport) {
+        const participants = lodashGet(report, 'participantAccountIDs', []);
+        if (!participants.includes(assigneeAccountID)) {
+            optimisticData[1].value.participantAccountIDs = [...participants, assigneeAccountID];
```

Comment:
Please extract the object referred to as `optimisticData[1].value` to a local variable like `const optimisticReport`, defined above `const optimisticData`.

Reply:
Thanks for the suggestion. I applied the changes. I hope the changes were what you wanted.